tests/test_logger.py

```diff
@@ -11,6 +11,11 @@
 from tests.basic_env import TestDvc
 
 
+class StringContaining(str):
```

Comment:
Don't know where to put that one. I don't see any test_utils (there actually is one, but it tests utils and it do not store test utils)

Reply:
This is fine as is :)

Reply:
This is fine as is :)